src/ReleaseApp/Domain/Entities/UpgradeInstructionModule.php

```diff
@@ -78,4 +78,24 @@ public function getType(): string
 
         return $this->body[static::TYPE_KEY];
     }
+
+    /**
+     * @param string $version
+     *
+     * @return void
+     */
+    public function setVersion(string $version): void
```

Comment:
Minor: would be fine to keep setter and getter of the one property together

Reply:
++